### DISCUSSION TITLE
Raster image import problem on wine (Ubuntu)

### DISCUSSION THREAD

**Discussion by Fanop11** [arkypita/LaserGRBL#1771] [Q&A]
Hi friend, importing raster image (jpeg) stuck on wine (Ubuntu).  I got the following on Playonlinux debugger:
0138:fixme:gdiplus:GdipSetImageAttributesThreshold not implemented.
![Screenshot_20220220_093932](https://user-images.githubusercontent.com/100060448/154832047-2ebd5432-7d34-4702-a9ee-36e03039beac.png)
Many thanks.

**Comment by cbagwell** [comment]
I see this as well on MacOS with wine as described in #1765 .  If you click on the 1bit BW Dithering or Passthrough options in screenshot then you should see the image.

The version of gdiplus that ships with wine is missing the needed features for those other options.  You can use winetricks or similar solution to install gdiplus and then it should work better for you.  The discussion for Linux in #1764 towards end gives instructions on steps to get working with Linux.

**Comment by archy587** [comment]
yes same problem! from that link:

Download latest winetricks from https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks 
$ chmod a+x winetricks 
$ ./winetricks gdiplus msxml3

the first attempt to install failed, but prompted me to install cabextract with $ sudo apt-get install cabextract

after that gdiplus installed correctly and im up and running on ubuntu 22.04 with wine 7.18 staging version

love laserGRBLs easy raster and gcode generation interface! so happy to have it working on linux after my only windows machine died.

thanks!
